downtimer/utils.py

```diff
@@ -15,9 +15,10 @@ def do_check(endpoint, address, db_adapter):
             r = requests.head(address, timeout=SERVICE_TIMEOUT)
             status_msg = 'FAIL'
             if r.status_code >= 400:
-                r = requests.get(address + 'healthcheck', timeout=SERVICE_TIMEOUT)
+                _address = address + 'healthcheck'
+                r = requests.get(_address, timeout=SERVICE_TIMEOUT)
                 if r.status_code < 300:
-                    address = address + 'healthcheck'
+                    address = _address
```

Comment:
Kevin, this line and 4 more lines above made this code a bit tricky. Could you please add small comment above this line clarifying things. For example, it can be something like "request to http://<service_endpoint>/<service_port> returned bad status code, so we're trying to do request to http://<service_endpoint>/<service_port>/healthcheck url"